src/Controls/src/Core/HybridWebView/PlatformHybridWebViewWebResourceRequestedEventArgs.cs

```diff
@@ -247,7 +182,10 @@ internal PlatformHybridWebViewWebResourceRequestedEventArgs(WebResourceRequested
 
 	internal string? GetRequestMethod() => Request.Method;
 
-	internal IReadOnlyDictionary<string, string>? GetRequestHeaders() => Request.RequestHeaders?.AsReadOnly();
+	internal IReadOnlyDictionary<string, string> GetRequestHeaders() =>
+		_headers ??= Request.RequestHeaders is { } rh
+			? new Dictionary<string, string>(rh, StringComparer.OrdinalIgnoreCase)
+			: new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
```

Comment:
Unfortunately, depending on the server and how the headers are populated, Android sometimes converts the headers to lower case. So we always will ned a case-insensitive.

Windows was already case insensitive, so this just makes it all consistent.